Magento_Theme/web/css/source/blocks/_navigation.less

```diff
@@ -15,6 +17,45 @@
     }
 
     .switcher-currency a:hover {
-        color: @color-orange-red4;
+        color: @color-vaimo-yellow;
+    }
+
+      .greet.welcome .not-logged-in {
+        display: none;
+    }
+
+    .nav-sections-item-title.active a {
+        color: @color-vaimo-yellow;
+    }
+
+    .navigation .level0.active > .level-top {
+        border-color: @color-vaimo-yellow !important;
+    }
+}
+
+
+//
+//  Desktop
+//  _____________________________________________
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
+    .switcher-option a {
+        color: #333;
+    }
+
+    .switcher-option a:hover {
+        color: white;
+        background-color: @color-vaimo-yellow;
+    }
+
+    .navigation .level0 .submenu a:hover {
+        background-color: @color-vaimo-yellow;
+        color: white;
```

Comment:
even when the color is simple like black, white but it has to be applied for the same logical blocks, it'd be better to use variables as well.
The reason is when a client ask you to replace the links color for the header and main navigation from white to another one, you could do that by the value changing only for variable, not to replace it everywhere by hardcoding

Reply:
a meant a variable for `white` as well